### DISCUSSION TITLE
Good vs. bad indicators

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#708] [Ideas]
Has anyone ever tried to determine what is a good versus bad indicator?

I may try at some point to add some kind of scientific method to make some assessments to make it part of our documentation.  It may not necessarily be a good/bad determination, but perhaps some kind of [heat map](https://github.com/kurkle/chartjs-chart-matrix) for optimal parameter arguments in different market conditions.

**Comment by tiger2014** [comment]
Hi Skender,

I am also thinking about this topic. I have a file that has a stock price and tagged up/down trend manually. 

I'm considering 2 things: 

1, which indicators should be applied to the stock price sequence (here is [a referring article](https://www.linkedin.com/pulse/building-live-strategy-works-tad-slaff/?trk=article-ssr-frontend-pulse_more-articles_related-content-card)); 

2, how to measure those indicators are more important for identifying a trend 

Hope you have a good idea.
